include/apps/clock/OswAppTimer.h

```diff
@@ -0,0 +1,60 @@
+#ifndef OSW_APP_TIMER_H
+#define OSW_APP_TIMER_H
+
+#include "../lib/date/date.h"
+
+#include "osw_app.h"
+#include <osw_hal.h>
+#include <osw_ui.h>
+#include "apps/main/switcher.h"
+#include "./services/NotifierClient.h"
+
+#include <array>
+
+class OswAppTimer : public OswApp
+{
+public:
+    enum class TimerState
+    {
+        IDLE,
+        SET_TIMER_SCREEN,
+        RUNNING,
+        PAUSED
+    };
+
+    OswAppTimer(OswAppSwitcher *clockAppSwitcher);
+    void setup() override;
+    void loop() override;
+    void stop() override;
+    ~OswAppTimer(){};
+
+private:
+    // Handle buttons
+    void handleNextButton(const unsigned char optionsCount);
+    void handleIncrementButton();
+    void handleDecrementButton();
+    long handleResetButton();
+
+    // Helpers
+    void timestampToSec();
+    void resetTimer();
+
+    // Draw on screen
+    void drawNumber(const int number, const int index);
+    void drawTime(const int totalSeconds);
+
+    OswAppSwitcher *clockAppSwitcher{};
+    NotifierClient notifierClient{"com.akmal-threepointsix.application.timer"};
```

Comment:
IIRC you plan to merge this into OSW itself -> maybe use `org.open-smartwatch.osw.timer` instead?